pom.xml

```diff
@@ -50,16 +50,15 @@
             <version>2.0</version>
         </dependency>
         <dependency>
+            <!-- https://github.com/jenkinsci/workflow-api-plugin/pull/39 -->
```

Comment:
Well the comment is irrelevant now that you are on a release version! So delete.